### ISSUE TITLE
Rule not found in Scirius, but present in EveBox/Suricata

### ISSUE THREAD

**Issue by skamoen** [StamusNetworks/scirius#117]
I'm running SELKS, and I'm getting alerts for SID  2523318 (ET TOR Known Tor Relay/Router (Not Exit) Node TCP Traffic group 660). Alerts for this rule show up in the graphs, but they don't show up under "Rules Activity" and a manual lookup at `/rules/rule/pk/2523318/` results in a 404. I'd like to supress this rule, but without access from Scirius that doesn't really seem possible. 

Scirius does have a lot of similar rules like "ET TOR Known Tor Relay/Router (Not Exit) Node TCP Traffic group xxx", but they go up to 624.

Any idea what could be going on here?

**Comment by regit** [comment]
Does you suricata.yaml only reference scirius.rules ? If not you may have some other rules loaded. Another possiblity is that a suricata rules update/restart is needed.

**Comment by skamoen** [comment]
I didn't touch the part on rules since installation, but there's a whole list of rule files in suricata.yaml. Should only scirius.rules be in there?

**Comment by pevma** [comment]
Yes - you should only have scirius.rules activated in the suricata.yaml
